DBM-Party-Vanilla/Classic/SOD_DemonFallCanyon/Diathorus.lua

```diff
@@ -8,20 +8,29 @@ mod:SetEncounterID(3024)
 mod:RegisterCombat("combat")
 
 mod:RegisterEventsInCombat(
---	"SPELL_CAST_START",
---	"SPELL_CAST_SUCCESS,
---	"SPELL_AURA_APPLIED"
+	"SPELL_CAST_START 460755",
+	"SPELL_AURA_APPLIED 460759"
 )
 
---function mod:OnCombatStart(delay)
+local specWarnGTFO 			= mod:NewSpecialWarningGTFO(460759, nil, nil, nil, 1, 8)
+local specWarnVeilOfShadow	= mod:NewSpecialWarningInterrupt(460755, "HasInterrupt", nil, nil, 1, 2)
 
---end
+-- Shadow Bolts can be kicked, but they are cast a lot, warnings would be very spammy
+-- "Shadow Bolt-460749-npc:227019-000012D5C6 = pull:42.6, 1.7, 2.0, 3.2, 11.3, 7.0, 2.7, 1.6, 1.6, 9.7, 1.6, 4.8, 1.6, 1.6, 1.7",
+
+-- Wowhead says to not stand in 460764, but I can't see that in my logs as a fire effect? Seems like a normal boss buff?
 
 
---[[
 function mod:SPELL_CAST_START(args)
-	if args:IsSpell(5174) then
+	if args:IsSpell(460755) then
+		specWarnVeilOfShadow:Play("kickcast")
+		specWarnVeilOfShadow:Show()
```

Comment:
This would show "interrupt unknown" without target name (which should be args.sourceName

Reply:
oops, forgot to commit after fixing this :)